posthog/tasks/test/test_usage_report.py

```diff
@@ -29,574 +30,512 @@
+                    "deployment_infrastructure": "tests",
+                    "realm": "hosted-clickhouse",
+                    "period": {
+                        "start_inclusive": "2022-01-09T00:00:00+00:00",
+                        "end_inclusive": "2022-01-09T23:59:59.999999+00:00",
+                    },
+                    "site_url": "http://test.posthog.com",
+                    "product": "open source",
+                    "helm": {},
+                    "clickhouse_version": all_reports[1]["clickhouse_version"],
+                    "users_who_logged_in": [],
+                    "users_who_logged_in_count": 0,
+                    "users_who_signed_up": [],
+                    "users_who_signed_up_count": 0,
+                    "table_sizes": all_reports[1]["table_sizes"],
+                    "plugins_installed": {"Installed and enabled": 1, "Installed but not enabled": 1},
+                    "plugins_enabled": {"Installed and enabled": 1},
+                    "event_count_lifetime": 10,
+                    "event_count_in_period": 10,
+                    "event_count_in_month": 10,
+                    "event_count_with_groups_in_period": 0,
+                    "recording_count_in_period": 0,
+                    "recording_count_total": 0,
+                    "group_types_total": 0,
+                    "dashboard_count": 0,
+                    "dashboard_template_count": 0,
+                    "dashboard_shared_count": 0,
+                    "dashboard_tagged_count": 0,
+                    "ff_count": 0,
+                    "ff_active_count": 0,
+                    "date": "2022-01-09",
+                    "organization_id": str(self.org_2.id),
+                    "organization_name": "Org 2",
+                    "organization_created_at": "2022-01-10T00:01:00+00:00",
+                    "organization_user_count": 0,
+                    "team_count": 1,
+                    "teams": {
+                        self.org_2_team_3.id: {
+                            "event_count_lifetime": 10,
+                            "event_count_in_period": 10,
+                            "event_count_in_month": 10,
+                            "event_count_with_groups_in_period": 0,
+                            "recording_count_in_period": 0,
+                            "recording_count_total": 0,
+                            "group_types_total": 0,
+                            "dashboard_count": 0,
+                            "dashboard_template_count": 0,
+                            "dashboard_shared_count": 0,
+                            "dashboard_tagged_count": 0,
+                            "ff_count": 0,
+                            "ff_active_count": 0,
+                        }
+                    },
+                },
+            ]
+
+            assert expectation == all_reports
+
+
+#     def test_status_report_duplicate_distinct_ids(self) -> None:
```

Comment:
What's up with all these tests?

Reply:
Some we might not be able to support anymore, others I just still need to do